realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -203,7 +203,7 @@ public boolean isValid() {
     public RealmQuery<E> isNull(String fieldName) {
         realm.checkIfValid();
 
-        long columnIndices[] = schema.getColumnIndices(fieldName);
+        long columnIndices[] = schema.parseFieldName(fieldName);
```

Comment:
is `parseFieldName` really a better name than `getColumnIndicies`? The former seemed more clear to me? `parsing` could mean any number of things?